content/millennium/dstu2/medications/medication-administration.md

```diff
@@ -74,21 +75,29 @@ _Implementation Notes_
 ### Parameters
 
 
- Name           | Required?          | Type          | Description
-----------------|--------------------|---------------|-----------------------------------------------------------------------------------------------
-`_id`           | This, or `patient` | [`token`]     | The logical resource id associated with the resource.
-`patient`       | This, or `_id`     | [`reference`] | The patient who has received the medication administration. Example: `patient=12345`
-`status`        | N                  | [`token`]     | The status of the medication administration, may be a list separated by commas.  Example: `status=completed`
-`practitioner`  | N                  | [`reference`] | The performing clinician, may be a list separated by commas. Example: `practitioner=12345`
-`notgiven`      | N                  | [`token`]     | Administrations that were not made. A value of 'true' will search for these, and a value of 'false' will exclude them. Example: `true`
-`effectivetime` | N                  | [`date`]      | The effectivetime search may be provided once with either the ge or le prefix or twice with the ge and le prefixes to indicate a specific range. Example: `effectivetime=ge2010-08-18` or `effectivetime=le2015-07-17` or `effectivetime=ge2010-08-18&effectivetime=le2015-07-17`
-[`_count`]      | N                  | [`number`]    | The maximum number of results to return.
-
+ Name           | Required?     | Type          | Description
+----------------|---------------|---------------|-----------------------------------------------------------------------------------------------
+`_id`           | Conditionally | [`token`]     | The logical resource id associated with the resource. It may be a list separated by commas. This parameter is required if the `patient` parameter is not used. Example: `_id=12345`
+`patient`       | Conditionally | [`reference`] | The patient who has received the medication administration. This parameter is required if the `_id` parameter is not used. Example: `patient=1234`
```

Comment:
capitalize Patient in the description?

Reply:
can remove 'has' as well

Reply:
Re-wording to match R4's better.  That one spoke it better anyways.